hack/schema/main.go

```diff
@@ -239,6 +239,12 @@ func getReflector() (*jsonschema.Reflector, error) {
 	}
 	r.CommentMap = commentMap
 
+	for k, comment := range commentMap {
+		if strings.Contains(comment, "<") || strings.Contains(comment, ">") {
```

Comment:
Nit: Long term might be better to check for more consistent strings. I think there is a Constant in golang core library, though this looks like special case.

Reply:
yeah, so if anyone puts a `<` or `>` in the comments [here](https://github.com/loft-sh/vcluster/blob/main/config/config.go), it breaks `.mdx` rendering (as we use comments as a source of content for field description)